scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -227,7 +227,10 @@ object BigQueryTable {
     private[bigquery] val DefaultTimePartitioning: TimePartitioning = null
     private[bigquery] val DefaultExtendedErrorInfo: ExtendedErrorInfo = ExtendedErrorInfo.Disabled
     private[bigquery] val DefaultInsertErrorTransform
-      : SCollection[DefaultExtendedErrorInfo.Info] => Unit = _ => ()
+      : SCollection[DefaultExtendedErrorInfo.Info] => Unit =
+      // A NoOp on the extended error output, so that we don't have DropInputs
+      // in the pipeline graph.
+      _.withName("Ignore ExtendedErrorInfo").map(_ => ())
```

Comment:
Trying to understand this. How is adding a transform getting rid of it from the graph?

Reply:
We have a step in Beam, where before running (submitting) the job to the runner, it is traversing the graph and consuming all transforms which are Reads without a Sink. This code is [here](https://github.com/apache/beam/blob/master/runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/UnconsumedReads.java)

The BQ write transform outputs errors by creating a read of insert errors.

This adds a [NoOp](https://github.com/apache/beam/blob/master/runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/UnconsumedReads.java#L67-L69) transform to consume this read.


When we organize the pipeline in multiple composable transforms, having a bq write from within a transform results in the `NoOp` lying outside of transform.

Also if we write to BQ, this transform shows up as [DropInput](https://github.com/apache/beam/blob/master/runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/UnconsumedReads.java#L63) which can be confusing for someone not very familiar with this.

Reply:
Might be hard to test plus a bit overkill. LGTM as is.